### PR TITLE
[WIP][NV-928] - Add cypress component test to OrganizationSelect component

### LINKED ISSUE #1457: [NV-928] - Add cypress component test to OrganizationSelect component
## Why?

To help increase our test coverage within the `novu/web` project, we can add the newly released [cypress component tests](https://docs.cypress.io/guides/component-testing/testing-react).

## How?

A new component test file should be added located near the original source file(see example at `apps/web/src/design-system/button/button.cy.tsx` and could be run after running `npm run start` from the root of the monorepo and specifying "Run Tests" and "Open Cypress", after which select the "Component testing" in the cypress screen.

Write the new test and make sure it passes all assertions.

The component is located at: apps/web/src/components/layout/components/OrganizationSelect.tsx

### PR DESCRIPTION
**Pull request by ludovicobesana**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...) Component Testing

- **Why was this change needed?** (You can also link to an open issue here) https://github.com/novuhq/novu/issues/1457
Close #1457 

- **Other information**:

### CODE REVIEW

**Comment by ludovicobesana** [comment]
@soundarrajan-official I've added some other tests, maybe we can work on this together

**Comment by ludovicobesana** [comment]
@scopsy  Did you see this?

File: apps/web/src/components/layout/components/OrganizationSelect.cy.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import OrganizationSelect from './OrganizationSelect';
+import { TestWrapper } from '../../../testing';
+import { Select } from '../../../design-system';
+
+describe('Organization Select Component Test', () => {
+  // Get the locator
+  const organizationSelectComponentSelector = '[data-test-id="organization-switch"]';
+  const queryClient = new QueryClient();
+
+  // TEST 1
```
Comment (p-fernandez) [inline apps/web/src/components/layout/components/OrganizationSelect.cy.tsx:11]: These comments doesn't provide much value. I would suggest you to remove them.

File: apps/web/src/components/layout/components/OrganizationSelect.cy.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+  // TEST 2
+  it('should get the organization-switch data test id', () => {
+    cy.mount(
+      <TestWrapper>
+        <QueryClientProvider client={queryClient}>
+          <OrganizationSelect />
+        </QueryClientProvider>
+      </TestWrapper>
+    );
```
Comment (p-fernandez) [inline apps/web/src/components/layout/components/OrganizationSelect.cy.tsx:30]: This test is missing the check. Like this has no difference with the render test.

**Review by p-fernandez — CHANGES_REQUESTED** [review]
This is a good one! Let us know when it is not in draft anymore and is ready for a final review.

**Comment by ludovicobesana** [comment]
@scopsy I'll work on this today, so we can close this Issue.
Thank you

**Comment by scopsy** [comment]
Closing the PR due to inactivity, @ludovicobesana feel free to reopen it once you are ready 🙏

**Pull request closed without merging**